trade-finance-manager-api/src/v1/api.js

```diff
@@ -254,6 +254,25 @@ const updateFacility = async (facilityId, facilityUpdate) => {
   }
 };
 
+const createFacilityAmendment = async (facilityId, amendmentsUpdate) => {
+  try {
+    const response = await axios({
+      method: 'put',
+      url: `${centralApiUrl}/v1/tfm/facilities/${facilityId}/amendments`,
```

Comment:
we are getting quite a few notifications about SSRF vulnerabilities. 
For this api call, can you validate the facilityId (use monodb for this) and centralApi url (you can probably use Joi for this)

Reply:
done